Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -39,15 +39,13 @@
 
 static const wchar_t* TAG = L"CGImage";
 
-// TODO #1124: remove old code
-#pragma region OLD_CODE
+// This is used by XamlCompositor to flush the DisplayTexture cache.
+// TODO GH#2098 look at where we're using the image cache and what we can do to avoid it.
 static std::vector<CGImageDestructionListener> _imageDestructionListeners;
```

Comment:
ok, then  's_' for clarity

---
In reply to: [103062541](https://github.com/Microsoft/WinObjC/pull/2099#discussion_r103062541) [](ancestors = 103062541)